### ISSUE TITLE
false clashes errors

### ISSUE THREAD

**Issue by brianmay** [django-mptt/django-mptt#375]
Since 0.7.1, I get errors like:

```
karaage.Project.level: (models.E006) The field 'level' clashes with the field 'level' from model 'karaage.project'.
karaage.Project.lft: (models.E006) The field 'lft' clashes with the field 'lft' from model 'karaage.project'.
karaage.Project.rght: (models.E006) The field 'rght' clashes with the field 'rght' from model 'karaage.project'.
karaage.Project.tree_id: (models.E006) The field 'tree_id' clashes with the field 'tree_id' from model 'karaage.project'.
```

However, it seems to be getting confused.

There is a Project model in the ``karaage.projects.models`` class, that belongs to the ``karaage`` application (ie. the meta value ``app_label`` is set to ``'karaage'``). So it seems to be saying it conflicts with itself.

This was fine in django-mptt 0.6.1

Thanks

**Comment by craigds** [comment]
This seems more likely to be caused by a django upgrade (probably 1.6 to 1.7) than a django-mptt upgrade. Did you upgrade django at the same time? What version of django are you using?

**Comment by brianmay** [comment]
That was my initial thought too.

However, I know Django 1.7 works fine with this project, and this error occurs with Django 1.7. If I downgrade mptt to 0.6.1 without making any changes, it works; similarly if I upgrade to 0.7.1 or 0.7.2 without making any other changes it stops working.

**Comment by craigds** [comment]
@brianmay does latest master fix it? The fix for #376 seems like it might be related.

**Comment by brianmay** [comment]
@craigds Yes, the master version does seem a lot healthier. Thanks.
